.github/workflows/frontend-ci.yml

```diff
@@ -0,0 +1,39 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+
+jobs:
+  build-frontend:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [18.x]
```

Comment:
@tuomasmustakallio @OttoAhlfors which node versions do we want to test?

Reply:
I'm using 20.11.0 but I think 18 works also

Reply:
Actually Node 20 LTS would probably be best on the short research(reddit thread) that I did. It has slightly better performance